### DISCUSSION TITLE
Use privilege escalation: become

### DISCUSSION THREAD

**Discussion by SckyzO** [bluebanquise/bluebanquise#507] [Ideas]
For security reasons it would be better to use privilege escalation: **become** rather than running playbooks as root

**Comment by btravouillon** [comment]
Hello @SckyzO ,

Thank you for your comment. The purpose of the stack is to provide Ansible roles. There are some examples of playbooks to help with the initial setup of a cluster of hosts, but we cannot cover all the use cases, nor the wide range of Ansible configuration, thus we use the easiest way to run playbooks to configure systems. Each sysadmin has to leverage to possibilities of Ansible to fulfill the needs of their organization.

I will convert this issue to a discussion though, because this can be useful to discuss this further with other users of our community. If some consensus is found on how one could implement security in their IT, we could consider adding a chapter about this in the documentation.

**Comment by oxedions** [comment]
Dear @SckyzO 

My apologies for the very late answer. I had my second child this month, and so it is difficult for me to keep up with everything 😊 Should be better in January.

I understand your need. What I can propose it at least to document how to run a playbook using become, like here: https://www.middlewareinventory.com/blog/ansible-sudo-ansible-become-example/

I however I do not know if everything will work using this. This need to be tested.

So a small section covering execution possibilities. This can be done in stories and linked from main installation documentation. We also needed to cover Ansible Vault in the same way.

Would that answer your need ?

Have a merry Christmas guys 🎄 

Ox
